### DISCUSSION TITLE
Welcome to Xedge32 Discussions!

### DISCUSSION THREAD

**Discussion by Telemechanics** [RealTimeLogic/Xedge32#16] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MajorLee95** [comment]
Hello

I try to follow the Getting Started page to flash Xedge32 in an ESP32-S3-DevKitC-1 N16R8, specially purchased for this occasion.

I succeded to flash the code and i switched the usb port to have the LuaShell32 prompt.

But in the note it is mentionned that Real Time Logic provide 2 firmwares : xedge.bin and xedge-s0.bin but in the downloaded zip file there is only one file xedge.bin. Where can we download xedge-s0.bin.

Thanks for your great job !

**Comment by surfskidude** [comment]
The documentation was updated one hour ago, along with uploading a new https://realtimelogic.com/downloads/bas/Xedge32-S3-Firmware.zip

Maybe you just have to download again and make sure the download is not cached.

**Reply by MajorLee95** [reply]
Nice
